eng/GenerateAnalyzerNuspec.targets

```diff
@@ -27,7 +27,7 @@
       <_GeneratedRulesetsDir>$(IntermediateOutputPath)Rulesets</_GeneratedRulesetsDir>
       <ContainsPortedFxCopRules Condition="'$(ContainsPortedFxCopRules)' == ''">false</ContainsPortedFxCopRules>
       <GeneratePackagePropsFile Condition="'$(GeneratePackagePropsFile)' == ''">true</GeneratePackagePropsFile>
-      <DevelopmentDependency Condition="'$(DevelopmentDependency)' == ''">true</DevelopmentDependency>
```

Comment:
Seems `DevelopmentDependency` got defaulted to `false` instead of empty somewhere in arcade's lifetime. We now explicitly force it true for all packages that have analyzer assembly or analyzer nuget package dependency.